### PR TITLE
Use SetNumberFormatDigitOptions

### PR DESCRIPTION
**Pull request by zbraniecki**
Use https://github.com/tc39/ecma402/pull/107 in Initialization of PluralRules

### CODE REVIEW

File: spec/pluralrules.html [hunk 1]
```diff
@@ -24,16 +24,11 @@
...
-        1. Let _minFD_ be Get(_options_, *"minimumFractionDigits"*).
-        1. Set _pluralRules_.[[MinimumFractionDigits]] to _minFD_.
-        1. Let _maxFD_ be Get(_options_, *"maximumFractionDigits"*).
-        1. Set _pluralRules_.[[MaximumFractionDigits]] to _maxFD_.
-        1. Let _minSD_ be Get(_options_, *"minimumSignificantDigits"*).
-        1. Set _pluralRules_.[[MinimumSignificantDigits]] to _minSD_.
-        1. Let _maxSD_ be Get(_options_, *"maximumSignificantDigits"*).
-        1. Set _pluralRules_.[[MaximumSignificantDigits]] to _maxSD_.
+        1. Call SetNumberFormatOptions(_pluralRules_, _options_).
+        1. If _pluralRules_.[[minimumIntegerDigits]], then
```
Comment (caridy) [inline spec/pluralrules.html]: we need a more explicit condition, I suspect that you're testing against `*undefined*` here.

File: spec/pluralrules.html [hunk 2]
```diff
@@ -24,16 +24,11 @@
...
+          1. Set _pluralRules_[[minimumIntegerDigits]] to *0*.
+        1. If _pluralRules_.[[maximumFractionDigits]], then
```
Comment (caridy) [inline spec/pluralrules.html]: same as above.

File: spec/pluralrules.html [hunk 3]
```diff
@@ -24,16 +24,11 @@
...
+          1. Set _pluralRules_[[maximumFractionDigits]] to *3*.
```
Comment (caridy) [inline spec/pluralrules.html]: is there any particular reason why this is set to 3 by default?

**Review by caridy — CHANGES_REQUESTED** [review]

File: spec/pluralrules.html (hunk 3 above)
Comment (zbraniecki) [reply spec/pluralrules.html]: it should be `max(_pluralRules_.[[minimumFractionDigits]], 3)`, same as the default in numberFormat. Fixing.

File: spec/pluralrules.html [hunk 4]
```diff
@@ -24,16 +24,11 @@
...
-        1. Set _pluralRules_.[[MinimumIntegerDigits]] to _minID_.
-        1. Let _minFD_ be Get(_options_, *"minimumFractionDigits"*).
-        1. Set _pluralRules_.[[MinimumFractionDigits]] to _minFD_.
-        1. Let _maxFD_ be Get(_options_, *"maximumFractionDigits"*).
-        1. Set _pluralRules_.[[MaximumFractionDigits]] to _maxFD_.
-        1. Let _minSD_ be Get(_options_, *"minimumSignificantDigits"*).
-        1. Set _pluralRules_.[[MinimumSignificantDigits]] to _minSD_.
-        1. Let _maxSD_ be Get(_options_, *"maximumSignificantDigits"*).
-        1. Set _pluralRules_.[[MaximumSignificantDigits]] to _maxSD_.
+        1. Call SetNumberFormatOptions(_pluralRules_, _options_).
```
Comment (caridy) [inline spec/pluralrules.html]: `1. Perform ! SetNumberFormatOptions(_pluralRules_, _options_).` since that abstract operation has assertions.

**Review by caridy — CHANGES_REQUESTED** [review]

**Comment by caridy** [comment]
sorry, didn't notice the other small issue with the call, don't use call anymore, use perform, and when calling an abstract make sure you use either ? or ! depending on what the abstract does.

**Comment by zbraniecki** [comment]
fixed in both :)

**Review by caridy — APPROVED** [review]

**Pull request merged**